Imaginea/AdaptiveCards
# Implemented the background image component to support stretch and repeat mode

Worked to resolve the issue: https://github.com/Imaginea/AdaptiveCards/issues/36

Steps to test the PR:
1. Launch the App.
2. Navigate to "BackgroungImage.RepeatHorizontally.json" from the visualiser.
3. Expected: The backgroundImage will repeat horizontally.

As same as above, we can test  RepeatVertically, Repeat and Stretch mode for backgroundImage.
